ledger-tool/src/main.rs

```diff
@@ -2523,6 +2530,22 @@ fn main() {
                     println!("  - data_len: {}", data_len);
                 }
             }
+
+            let print_total_accounts_stats = !arg_matches.is_present("no_total_accounts_stats");
+            if print_total_accounts_stats {
+                println!("Getting total accounts stats...");
+                let (total_accounts_stats, measure) = Measure::this(
+                    bank::get_total_accounts_stats,
+                    &bank,
+                    "getting total accounts stats",
+                );
```

Comment:
If we implement this way, won't this be a second complete accounts scan?
Can we refactor to get this information from the first scan in L2505? Perhaps a new bank method that returns all accounts and the `total_accounts_stats`?

Reply:
Ummm good question! Yes, it is a second accounts scan. I see there are slight differences in the two. Let me look into this more and if it matters for the stats I'm adding. I'll also look into a new bank method that returns all the accounts and the `total_accounts_stats` too.

Reply:
Done! I've collapsed the second scan into the first. Since there's only one loop now, I removed the flag for "no print total accounts stats", since that printing only happens once at the end.